tools/mdm/migration/mdmproxy/mdmproxy.go

```diff
@@ -238,12 +276,29 @@ func makeExistingProxy(existingURL, existingDNSName string) *httputil.ReversePro
 	return proxy
 }
 
-func makeFleetProxy(fleetURL string) *httputil.ReverseProxy {
+func makeFleetProxy(fleetURL string, debug bool) *httputil.ReverseProxy {
 	targetURL, err := url.Parse(fleetURL)
 	if err != nil {
 		panic("failed to parse fleet-url: " + err.Error())
 	}
 	proxy := httputil.NewSingleHostReverseProxy(targetURL)
+	if debug {
+		proxy.ModifyResponse = func(r *http.Response) error {
+			b, err := io.ReadAll(r.Body)
+			if err != nil {
+				return err
+			}
+			err = r.Body.Close()
+			if err != nil {
+				return err
+			}
+			r.Body = io.NopCloser(bytes.NewReader(b))
+
+			log.Println("Fleet response: ", string(b))
```

Comment:
should we print any extra information here? worried about the logs being harder to distinguish with multiple concurrent requests.

guessing this is just for staging testing with very few devices so not a concern, feel free to ignore if so